performance_tests/README.rst

```diff
@@ -0,0 +1,232 @@
+#####################################
+aws-encryption-sdk performance tests
+#####################################
+
+This module runs performance tests for the `AWS Encryption SDK Python`_.
+
+********
+Overview
+********
+
+This module tests the following keyrings / master key-providers:
+
+1. KMS Keyring / KMS Master Key Provider
+2. Raw AES Keyring / AES Master Key Provider
+3. Raw RSA Keyring / RSA Master Key Provider
+4. Hierarchy Keyring
+5. Caching CMM
+
+For each test on the above keyrings / master key-providers, this package measures:
```

Comment:
```suggestion
For each test on the above keyrings / master key providers, this package measures:
```

It's just "master key provider", no hyphen: https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/migrate-mkps-v2.html

Reply:
Done. Thanks